### PR TITLE
Add styling to MovieCard.

### LINKED ISSUE #18: Styling MovieCards to display movie info in a presentable format.
**Is your feature request related to a problem? Please describe.**
Add styling to MovieCards.

**Describe the solution you'd like**
Adding new classes for styling, use flexbox to format content in MovieCard.

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Added CSS animation to MovieCard when hovering and on mouse leave.
2. Added React hook to apply classes dynamically.

## Purpose
Make MovieCard presentable.

## Approach
Used keyframe animations on title-overlay and card divs.
Closes #18

### CODE REVIEW

**Review by Lap343 — DISMISSED** [review]
Its ok I guess

File: movie-app/src/Components/MovieCard.js [hunk 1]
```diff
@@ -1,14 +1,34 @@
+
+import {useState} from 'react';
+
 const MovieCard = ({movie}) => {
+  const moviePoster = {
+    backgroundImage: `url(${movie.Poster})`,
+  }
+
+  const [cardHover, setCardHover] = useState(false);
+// when true, apply show info animation
```
Comment (krferrell) [inline movie-app/src/Components/MovieCard.js]: comment here

File: movie-app/src/Components/MovieCard.js [hunk 2]
```diff
@@ -1,14 +1,34 @@
...
+  const showInfoOnHover = () => {
+    setCardHover(!cardHover)
+  }
+
+  const hideInfoOnHover = () => {
+    setCardHover(!cardHover)
+  }
```
Comment (kmausolf) [inline movie-app/src/Components/MovieCard.js]: Both of these functions are the same.  You can combine them as a `toggleInfoOnHover` function.

File: movie-app/src/Components/MovieCard.js [hunk 3]
```diff
@@ -1,14 +1,34 @@
...
-      <h1>{movie.Title}</h1>
-      <h3>{movie.Year}</h3>
-    </div>
-  );
-};
+    <div className="card" style={moviePoster} onMouseEnter ={showInfoOnHover} onMouseLeave={hideInfoOnHover}>
 
+      <div className={`info-container ${cardHover? `hide-info`: `show-info`}`}>
+        <h2>{movie.Title}</h2>
+        <h3>{movie.Year}</h3>
```
Comment (kmausolf) [inline movie-app/src/Components/MovieCard.js:18]: These two can have the same heading level, as they're below an `h1` and Year is not necessarily a subsection of Title.

File: movie-app/src/body/Searched.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import MovieCard from '../components/MovieCard'
+import MovieCard from '../Components/MovieCard'
```
Comment (kmausolf) [inline movie-app/src/body/Searched.js:1]: The previous naming scheme for the `components` folder is more consistent with React standards.  Generally, the only thing that should start with an upper case letter is an actual component.

**Review by krferrell — DISMISSED** [review]
Looks decent

**Review by kmausolf — COMMENTED** [review]
Good focus in this PR.  I've left some feedback that may be helpful.

File: movie-app/src/Components/MovieCard.js (hunk 1 above)
Comment (kmausolf) [reply movie-app/src/Components/MovieCard.js]: This is a documentation comment.  Those are great!  What we don't want is old commented out code hanging around in our files that we don't plan to use.

**Comment by kmausolf** [comment]
@connielion I think you should add those comments back in.  They document the code and are helpful to other developers working on your code.  As I mentioned in my comment to Kenny, comments are great!  Old commented out code we don't plan to use is not good.

**Comment by kmausolf** [comment]
> Its ok I guess

I would like to ask that everyone takes the code review process seriously.  This is something that you will need to do for your team and for other teams all the time in the workplace, and the person who requested your review is trusting you to catch issues that they might have missed during development.

**Review by krferrell — APPROVED** [review]
Awesome work. Made some good progress with the styling!

**Pull request merged**